crypto/x509v3/pcy_cache.c

```diff
@@ -53,15 +57,19 @@ static int policy_cache_create(X509 *x,
         } else if (sk_X509_POLICY_DATA_find(cache->data, data) != -1) {
             ret = -1;
             goto bad_policy;
-        } else if (!sk_X509_POLICY_DATA_push(cache->data, data))
+        } else if (!sk_X509_POLICY_DATA_push(cache->data, data)) {
+            X509V3err(X509V3_F_POLICY_CACHE_CREATE, ERR_R_MALLOC_FAILURE);
             goto bad_policy;
+        }
         data = NULL;
     }
     ret = 1;
+
  bad_policy:
     if (ret == -1)
         x->ex_flags |= EXFLAG_INVALID_POLICY;
     policy_data_free(data);
+ fail_policy:
```

Comment:
The label `fail_policy` should be renamed to `err`, which is the standard cleanup label.